react/src/components/SummaryPageItems/SummaryItemShortCut.tsx

```diff
@@ -0,0 +1,186 @@
+import {
+  FileOutlined,
+  InboxOutlined,
+  SettingOutlined,
+  ToolOutlined,
+} from '@ant-design/icons';
+import { Typography, theme } from 'antd';
+import _ from 'lodash';
+import { useEffect, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface itemProps {
+  id: number;
+  title: string;
+  content: string;
+  icon?: JSX.Element;
+  onClick?: () => void;
+}
+
+const SummaryItemShortCut: React.FC = () => {
+  const { t } = useTranslation();
+  const { token } = theme.useToken();
+  const webuiNavigate = useWebUINavigate();
+  // In case of adding new items, implemented with useState
+  const [items] = useState<itemProps[]>([
+    {
+      id: 1,
+      title: t('summary.ChangeSystemSetting'),
+      content:
+        '계산환경 이미지, 그래픽 환경, 스케일링 및 플러그인을 변경합니다. ',
+      icon: (
+        <SettingOutlined
+          style={{
+            fontSize: token.fontSizeHeading3,
+            color: token.colorTextDescription,
+          }}
+        />
+      ),
+      onClick: () => {
+        webuiNavigate('/settings');
+      },
+    },
+    {
+      id: 2,
+      title: '사용자 설정 변경',
+      content: '개인 설정을 변경하여 사용자의 경험을 맞춤화할 수 있습니다.',
+      icon: (
+        <SettingOutlined
+          style={{
+            fontSize: token.fontSizeHeading3,
+            color: token.colorTextDescription,
+          }}
+        />
+      ),
+      onClick: () => {
+        webuiNavigate('/usersettings');
+      },
+    },
+    {
+      id: 3,
+      title: '자원 확인',
+      content: '현재 사용중인 자원을 확인하고 관리할 수 있습니다.',
+      icon: (
+        <InboxOutlined
+          style={{
+            fontSize: token.fontSizeHeading3,
+            color: token.colorTextDescription,
+          }}
+        />
+      ),
+      onClick: () => {
+        webuiNavigate('/agent');
+      },
+    },
+    {
+      id: 4,
+      title: '실행 환경 이미지 관리',
+      content: '실행 환경 이미지를 설치하거나 이미지 설정을 변경합니다.',
+      icon: (
+        <FileOutlined
+          style={{
+            fontSize: token.fontSizeHeading3,
+            color: token.colorTextDescription,
+          }}
+        />
+      ),
+      onClick: () => {
+        webuiNavigate('/environment');
+      },
+    },
+    {
+      id: 5,
+      title: '시스템 유지보수',
+      content: '시스템 유지보수를 위한 기능을 제공합니다.',
+      icon: (
+        <ToolOutlined
+          style={{
+            fontSize: token.fontSizeHeading3,
+            color: token.colorTextDescription,
+          }}
+        />
+      ),
+      onClick: () => {
+        webuiNavigate('/maintenance');
+      },
+    },
+  ]);
+  const [chunkSize, setChunkSize] = useState<number>(2);
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const container = containerRef.current;
+    if (!container) {
+      return;
+    }
+    const updateImageSize = () => {
+      const containerWidth = container.clientWidth;
+
+      if (containerWidth <= 410) {
+        setChunkSize(1);
+      } else if (containerWidth <= 820) {
```

Comment:
How about replacing it with token's screen size?

Reply:
The screen size should be based on the screen size of the page, but BAIBoard doesn't provide a ref property. 

And the size of the screen size token is too large to use.

Reply:
Maybe we can set "screenXS" value to 410.

Reply:
or we can just create custom-token for that.
https://ant-design.github.io/antd-style/guide/custom-theme#custom-token

Reply:
I applied it using antd's custom token.